docs/development/getting-started.rst

```diff
@@ -148,6 +148,11 @@ run all of the needed services. The Warehouse repository will be mounted inside
 the Docker container at :file:`/opt/warehouse/src/`. After the initial build,
 you should not have to run this command again.
 
+.. note::
+
+   On some systems ``make build`` may require the prior installation of libpq
+   and its headers so that psycopg2 can be compiled and installed.
+
```

Comment:
This seems unrelated?

Reply:
No, `make build` failed quickly, while installing requirements, and couldn't succeed until I installed libpq-dev on my laptop.

Reply:
Got it, sorry, didn't realize that you were planning to include this in this PR as well.